internal/keepers/factory.go

```diff
@@ -15,10 +15,11 @@ const maxObservationLength = 1_000
 var _ types.ReportingPluginFactory = (*keepersReportingFactory)(nil)
 
 type ReportingFactoryConfig struct {
-	CacheExpiration       time.Duration
-	CacheEvictionInterval time.Duration
-	MaxServiceWorkers     int
-	ServiceQueueLength    int
+	SamplingUpkeepJobDuration time.Duration
```

Comment:
This should be in the off chain config instead of the reporting factory config. The upkeep job duration is a configuration we want to control across the entire network.